entrypoint.py

```diff
@@ -73,16 +74,62 @@ async def pub_handler(request):
     if "payload" not in cmd["packet"]:
         cmd["packet"]["payload"] = ""
 
-    tm = a0.TopicManager(container = cmd["container"])
+    tm = a0.TopicManager(container=cmd["container"])
 
     p = a0.Publisher(tm.publisher_topic(cmd["topic"]))
-    p.pub(a0.Packet(
-        cmd["packet"]["headers"],
-        base64.b64decode(cmd["packet"]["payload"])))
+    p.pub(
+        a0.Packet(cmd["packet"]["headers"], base64.b64decode(cmd["packet"]["payload"]))
+    )
 
     return web.Response(text="success")
 
 
+# ws = new WebSocket(`ws://${api_addr}/api/pub`)
+# ws.onopen = () => {
+#     ws.send(JSON.stringify({
+#         container: "foo",
+#         topic: "bar",
+#     }))
+# }
+# // then:
+# ws.send(JSON.stringify({
+#         packet: {
+#             headers: [
+#                 ["key", "val"],
+#                 // ...
```

Comment:
please remove the `//`

Reply:
But the `//` makes it so that you can use this example with fewer changes.

Reply:
All of the `...` need to be replaced to work. Same for payload. More so for, headers. `{key: "val"}` is likely not what you want to use.

There is an expectation that anyone reading the code should be able to interpret and not use it directly.